documentation/List-of-Action-plugins.md

```diff
@@ -217,6 +217,16 @@ When an applicant re-submits an application after making changes, this Action up
 | ---------------------------------------- | ----------------- |
 | `applicationId`\*                        | `updatedReviews`  |
 
+### Clean Up Files
+
+After application submission or re-submission, this Action looks at all the uploaded files associated with the application and deletes any that are not part of a submitted response. This takes care of any files that the user "deleted" from the UI before submitting.
+
+- _Action Code:_ **`cleanupFiles`**
+
+| Input parameters<br />(\*required) <br/> | Output properties |
+| ---------------------------------------- | ----------------- |
+| `applicationSerial`                      | `deletedFiles`    |
+
```

Comment:
Nice - is this one already running for all applications that are using core actions?

Reply:
Yes, I think so.